agent/handlers/v2/response.go

```diff
@@ -272,3 +288,34 @@ func NewContainerResponse(
 	resp.Volumes = v1.NewVolumesResponse(dockerContainer)
 	return resp
 }
+
+// metadataErrorHandling writes an error to the logger, and append an error response
+// to V4 metadata endpoint task response
+func metadataErrorHandling(resp *TaskResponse, err error, field, resourceARN string, includeV4Metadata bool) {
+	seelog.Errorf("V2 response: unable to get '%s' for '%s': %s", field, resourceARN, err.Error())
```

Comment:
nit: since this function is used by v3 and v4 task metadata endpoints, it might be confusing to prepend the log with "V2 response". Maybe something more accurate like "Task Metadata error:"

Reply:
Update the error log message as suggested. Thanks! 
```
level=info time=xxx msg="V4 taskMetadata handler: Writing response for task 'taskARN'" module=task_metadata_handler.go
level=error time=xxx msg="Task Metadata error: unable to get 'ContainerInstanceTags' for 'containerInstanceARN': AccessDeniedException: User: iamARN is not authorized to perform: ecs:ListTagsForResource on resource: containerInstanceARN" module=response.go
level=error time=xxx msg="Task Metadata error: unable to get 'TaskTags' for 'taskARN': AccessDeniedException: User: iamARN is not authorized to perform: ecs:ListTagsForResource on resource: taskARN" module=response.go
```